signac/core/h5store.py

```diff
@@ -507,8 +507,8 @@ class H5StoreManager(DictManager):
     :param prefix:
         The directory prefix shared by all stores managed by this class.
     """
-    cls = H5Store
-    suffix = '.h5'
+    cls = H5Store  # type: ignore
+    suffix = '.h5'  # type: ignore
```

Comment:
Explanation for the ignore: These are overriding a default of `None` in the `DictManager` parent class, meaning that the type is changing. That raises an error.